### PR TITLE
Agorbunova/array methods

### PR DESCRIPTION
**Pull request by GorbunovaAnna**

### CODE REVIEW

File: Anna_Gorbunova/arrayMethods/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// 1. 
+// Пользователь вводит имя, если это имя есть в массиве1 добавьте его копию в массив2
+
+// let people1 = ['Samuel', 'Jack', 'Thomas', 'Henry', 'Leo', 'Connor', 'David', 'Ryan'];
+// let people2 = ['Connor', 'Stanley', 'Leo', 'Albert', 'Owen', 'Oliver', 'Ethan', 'Thomas'];
+
+// const userName = prompt('Enter name', '');
+
+// for (let i = 0; i < people1.length; i++) {
+//     if (userName === people1[i]) {
```
Comment (fashion-prepod) [inline Anna_Gorbunova/arrayMethods/index.js:10]: .indexOf()

File: Anna_Gorbunova/arrayMethods/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+// 4. 
+// Объедините два массива так, чтобы в получившемся массиве не было одинаковых имен
+
+// let people1 = ['Samuel', 'Jack', 'Thomas', 'Henry', 'Leo', 'Connor', 'David', 'Ryan'];
+// let people2 = ['Connor', 'Stanley', 'Leo', 'Albert', 'Owen', 'Oliver', 'Ethan', 'Thomas'];
+// let newArr = [];
+
+// for (let i = 0; i < people1.length; i++) {
+//     if (people2.includes(people1[i])) {
```
Comment (fashion-prepod) [inline Anna_Gorbunova/arrayMethods/index.js:56]:
```
if ( !people2.includes(people1[i]) ) {
  newArr.push(people1[i]);
}
```

**Review by fashion-prepod — CHANGES_REQUESTED** [review]

File: Anna_Gorbunova/arrayMethods/index.js (hunk 1 above)
Comment (GorbunovaAnna) [reply Anna_Gorbunova/arrayMethods/index.js]: fixed

File: Anna_Gorbunova/arrayMethods/index.js (hunk 2 above)
Comment (GorbunovaAnna) [reply Anna_Gorbunova/arrayMethods/index.js]: fixed

**Pull request merged**